tests/js/spec/views/issueList/overview.spec.jsx

```diff
@@ -1,16 +1,25 @@
+import {createRef} from 'react';
 import {browserHistory} from 'react-router';
-import cloneDeep from 'lodash/cloneDeep';
-import range from 'lodash/range';
+import * as qs from 'query-string';
 
-import {mountWithTheme, shallow} from 'sentry-test/enzyme';
 import {initializeOrg} from 'sentry-test/initializeOrg';
+import {fireEvent, mountWithTheme, waitFor} from 'sentry-test/reactTestingLibrary';
 
 import StreamGroup from 'app/components/stream/group';
-import GroupStore from 'app/stores/groupStore';
 import TagStore from 'app/stores/tagStore';
-import * as parseLinkHeader from 'app/utils/parseLinkHeader';
+import IssueListActions from 'app/views/issueList/actions';
 import IssueListWithStores, {IssueListOverview} from 'app/views/issueList/overview';
 
+// Taken from https://stackoverflow.com/a/56859650/1015027
+const withMarkup = query => text =>
```

Comment:
we you can reuse the function `import {findByTextContent} from 'sentry-test/utils'; `here